src/content/docs/journey-migration/migrating-logs.mdx

```diff
@@ -0,0 +1,82 @@
+
+<img
+    title="log-dashboard"
+    alt="An image displaying New Relic's log monitoring dashboard"
+    src={journeyLogsDash}
+/>
+
+## Your log monitoring solution [#logs-solution]
+
+Logs in the New Relic UI are found under the **Logs** section, exactly like you're used to in Datadog. New Relic log monitoring makes getting what you need from your logs as easy as possible, helping you avoid the headache that comes from parsing massive amounts of data. 
+
+You can automatically pinpoints events, errors, traces, and spans among your connected entities with just a single click, keeping you in one place and getting you to what you're looking for that much faster. Our log monitoring solution:
+
+* Enables deep visibility into your entity data, letting you see how your system works together.
+* Helps reduce your mean time to resolution (MTTR), letting you respond to issues faster.
+* Gives you the ability to create curated charts, dashboards, and alerts, giving you as detailed a look into your data as you want.
+* Provides the ability to visualize, search, and troubleshoot performance issues in one central location without the need to switch between tools.
+
+## Get started with New Relic log monitoring [#get-started]
+
+<SideBySide>
+    <Side>
+<img
+    title="Log monitoring installations"
+    alt="A screenshot displaying some of New Relic's log monitoring guided installs"
+    src={journeyLogInstall}
+/>
+    </Side>
+    <Side>
+    Once installed, your log monitoring is done in the **Logs** section of the UI, exactly like it is in Datadog. You can use our guided installations for logs after setting up any of our [infrastructure](/docs/journey-migration/migrating-infra/) or [APM](/docs/journey-migration/migrating-apm/) agents. 
+    
+    The steps for setting up logs varies depending on the data source, but no matter which you choose, you can rest assured knowing we'll guide you through every step of the process to make it as easy and straightforward as possible. You can see our list of guided installations for logs by selecting **Add Data > Logging** in the New Relic UI, or by clicking here.**MISSING DEEP LINK**
+    </Side>
+</SideBySide>
+
+
+## Explore your premade dashboard [#add-dashboard]
+
+New Relic's log monitoring gives you a premade central location with a variety of information available from the moment you set up the integration. No setup time required: just install the monitor and begin finding the data you need quickly.
+
+<img
+    title="Premade dashboards"
+    alt="An image displaying some of New Relic's premade infrastructure dashboards"
+    src={journeyLogsDash2}
+/>
+
+Our log dashboard automatically gives you access to:
```

Comment:
> Our log dashboard automatically gives you access to:

This feels like it's overselling something. Suggest simply:

> The logs UI has four key sections:

Or something similarly unadorned. Let your details and specific use case insight sell the feature. "automatically gives" isn't what will convince them

Reply:
Fixed!